10981.0.txt

```diff
@@ -0,0 +1,38 @@
+@0
+
+
+@1
+MOV UP, RIGHT
+#INPUT.A RIGHT
```

Comment:
When commenting individual lines of code, it's conventional to put the comment either above or to the right of the line